tests/kafkatest/services/mirror_maker.py

```diff
@@ -133,12 +145,19 @@ def start_node(self, node):
         node.account.ssh("mkdir -p %s" % MirrorMaker.LOG_DIR, allow_fail=False)
 
         # Create, upload one consumer config file for source cluster
-        consumer_props = self.render('consumer.properties', zookeeper_connect=self.source.zk.connect_setting())
+        if self.new_consumer:
+            consumer_props = self.render("new_consumer.properties", bootstrap_servers=self.source.bootstrap_servers())
+        else:
+            consumer_props = self.render("consumer.properties", zookeeper_connect=self.source.zk.connect_setting())
+            consumer_props += "\noffsets.storage=%s\n" % self.offsets_storage
+
         node.account.create_file(MirrorMaker.CONSUMER_CONFIG, consumer_props)
+        self.logger.info("Mirromaker consumer props:\n" + consumer_props)
```

Comment:
At the moment most of the services log this at the info level, so I'll probably stick with info for consistency... Also, this seems to generally be the convention in Kafka - brokers, consumers, etc etc log their configs at info level when they first start up.